llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -85,12 +85,14 @@ def IREELinalgExt_ScatterOp : IREELinalgExt_Op<"scatter",
   let arguments = (ins
       Variadic<AnyRankedTensorOrMemRefType>:$inputs,
       Variadic<AnyRankedTensorOrMemRefType>:$outputs,
+      I64ElementsAttr:$dimension_map,
```

Comment:
update the description section. I actually don't know if some of context are outdated or not. Updating the outdated part is also welcome.

Reply:
Added the portion about the dimension map. I don't think there is any part out of date but the doc is a little tricky to parse.